pyproject.toml

```diff
@@ -81,6 +90,7 @@ types-freezegun = "^1.1.0"
 types-chardet = "^0.1.5"
 types-certifi = "^0.1.4"
 types-aiofiles = "^0.1.9"
+Jinja2 = "^3.0.1"
```

Comment:
What is this new dependency for?

Reply:
no idea, I think there’s something broken with poetry :D will revert when it works again :)